cdci_osa_plugin/osa_common_pars.py

```diff
@@ -84,6 +84,31 @@ def get_osa_versions():
     return r_j
 
 
+class OSAVersion(Name):
+    @property
+    def value(self):
+        return self._value
+
+    @value.setter
+    def value(self,v):
+        if v is not None:
+            osa_version_base, osa_subversion, version_modifiers = split_osa_version(v)
+            # if self.check_value is not None:
+            #     self.check_value(v, units=self.units,name=self.name)
+            # if self._allowed_values is not None:
+            #     if v not in self._allowed_values:
+            #         raise RuntimeError('value',v,'not allowed, allowed=',self._allowed_values)
+
+            if osa_version_base not in ["OSA10.2", "OSA11.0", "OSA11.1"]:
+                raise RuntimeError('value', v, 'not allowed, allowed=', self._allowed_values)
+
+            if isinstance(v, (str, six.string_types)):
+                self._value = v.strip()
+            else:
+                self._value = v
```

Comment:
Do we allow different types to be set? Like a numeric one for example.

Reply:
No